src/patterns/news/examples/tag/index.njk

```diff
@@ -0,0 +1,202 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+{% from "components/question/_macro.njk" import onsQuestion %}
+{% from "components/radios/_macro.njk" import onsRadios %}
+{% from "components/textarea/_macro.njk" import onsTextarea %}
+{% from "components/input/_macro.njk" import onsInput %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/lists/_macro.njk" import onsList %}
+{% from "components/articles/_macro.njk" import onsArticles %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+{% set pageConfig = {
+    "title": 'Service name',
+    "absoluteUrl": pageInfo.url,
+    "pageColNumber": '12',
+    "breadcrumbs": {
+        "ariaLabel": 'Breadcrumb',
+        "itemsList": [
+            {
+                "url": '#0',
+                "text": 'Home'
+            },
+            {
+                "url": '#0',
+                "text": 'News'
+            }
+        ]
+    },
+    "footer": {}
+} %}
+
+{% block main %}
+
+<div class="grid">
+    <div class="grid__col col-7@m">
+        <h1 class="u-fs-xxl">All posts for Census2021</h1>
```

Comment:
I think it would help if we wrapped the tag word in a highlight span here too so the connection between the tag elements is clear

Reply:
Done